### PR TITLE
[fix] cdc data timestamp now drawn from source 

### PR DESCRIPTION
**Pull request by BriceWolfgang**

### CODE REVIEW

File: covid/extract.py
Comment (lucasmbrown) [inline covid/extract.py:52]: Very nice!

**Review by lucasmbrown — APPROVED** [review]
@patricksheehan should approve before we merge to see if this meets his needs, but LGTM!

File: covid/extract.py
Comment (patricksheehan) [inline covid/extract.py:61]: 💯

**Review by patricksheehan — APPROVED** [review]
LGTM, thanks for the quick fix!

**Pull request merged**
